pkg/kf/spaces/kfspace.go

```diff
@@ -116,6 +116,21 @@ func (k *KfSpace) ResetServices() {
 	delete(k.Spec.ResourceLimits.SpaceQuota, v1.ResourceServices)
 }
 
+// GetDomains gets the domains for the space.
+func (k *KfSpace) GetDomains() []string {
```

Comment:
Along with these, WDYT about adding config_space commands in this PR? https://github.com/google/kf/blob/master/pkg/kf/commands/spaces/config_space.go